sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -132,6 +134,9 @@ def __init__(
         self.checkpoint_enabled = checkpoint_enabled
         self.extra: dict = {}
 
+        if (self.gpu != "" or len(self.gpu) > 0) and self.gpu_count == 0:
```

Comment:
Have we tested this when `gpu` is a List?

Reply:
Will confirm